src/Z.EntityFramework.Plus.EFCore3x.NETStandard21/Z.EntityFramework.Plus.EFCore3x.NETStandard21.csproj

```diff
@@ -13,11 +13,13 @@
     <RootNamespace>Z.EntityFramework.Plus.EFCore.NETStandard21</RootNamespace>
     <SignAssembly>false</SignAssembly>
     <AssemblyOriginatorKeyFile>zzzproject.pfx</AssemblyOriginatorKeyFile>
-    <PackageId>Z.EntityFramework.Plus.EFCore</PackageId>
+    <PackageId>Interfirt.Z.EntityFramework.Plus.EFCore</PackageId>
     <Version>3.0.40</Version>
```

Comment:
use please version according to current sprint

Reply:
In this case I think we can add initial version like `<PackageReleaseNotes>Version 3.0.40</PackageReleaseNotes>`

Reply:
I disagree with version comment, that's external package and you need to know at what version it based on

Reply:
This is our (IM) package. And version related to our inner versioning. If you want to notify about smth you can do this with PackageReleaseNotes